### PR TITLE
feat: Update init command to return JSON

### PR DESCRIPTION
**Pull request by torchello**

### CODE REVIEW

File: lib/appmap/command/agent_setup/init.rb [hunk 1]
```diff
@@ -11,32 +10,19 @@ module AgentSetup
...
-            :green
-          )
-          puts content
-        end
-
-        private
+          result = {
+            :configuration => {
+              :filename => config_file,
+              :contents => YAML.dump(config).gsub("---\n", '')
```
Comment (apotterri) [inline lib/appmap/command/agent_setup/init.rb:21]: Why are you stripping out the separator(s)?

File: lib/appmap/command/agent_setup/init.rb [hunk 2]
```diff
@@ -11,32 +10,19 @@ module AgentSetup
...
-          puts AppMap::Util.color(
-            %(The following AppMap config file #{config_file} has been created:),
-            :green
-          )
-          puts content
-        end
-
-        private
+          result = {
+            :configuration => {
```
Comment (apotterri) [inline lib/appmap/command/agent_setup/init.rb:19]: The rest of appmap-ruby uses new-style syntax for hashes with symbol keys. Probably best to follow that convention here.

**Review by apotterri — CHANGES_REQUESTED** [review]

File: lib/appmap/command/agent_setup/init.rb (hunk 1 above)
Comment (torchello) [reply lib/appmap/command/agent_setup/init.rb]: We won't have multiple "documents" in one yaml, so it does not make sense here. Would you prefer to keep it?

Comment (apotterri) [reply lib/appmap/command/agent_setup/init.rb]: Unless the separators actually cause a problem, I think we should keep them.

**Review by apotterri — CHANGES_REQUESTED** [review]
Updated hashes look good, let's just get rid of the `gsub` that strips out  the separators.

File: lib/appmap/command/agent_setup/init.rb (hunk 1 above)
Comment (torchello) [reply lib/appmap/command/agent_setup/init.rb]: Removed!

**Review by apotterri — APPROVED** [review]

**Pull request merged**